WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -54,6 +56,7 @@ public SettingsViewModel() : base("Settings")
 			this.ValidateProperty(x => x.BitcoinP2pEndPoint, ValidateBitcoinP2pEndPoint);
 
 			Autocopy = Global.UiConfig.Autocopy;
+			FeeDisplayFormat = (FeeDisplayFormat)(Enum.ToObject(typeof(FeeDisplayFormat), Global.UiConfig?.FeeDisplayFormat) ?? FeeDisplayFormat.SatoshiPerByte);
```

Comment:
`Global.UiConfig` is always non-null, isn't it? If so, then:

```suggestion
			FeeDisplayFormat = (FeeDisplayFormat)(Enum.ToObject(typeof(FeeDisplayFormat), Global.UiConfig.FeeDisplayFormat) ?? FeeDisplayFormat.SatoshiPerByte);
```